x-pack/plugins/observability_solution/observability/public/pages/alert_details/alert_details_contextual_insights.tsx

```diff
@@ -99,21 +101,29 @@ ${JSON.stringify(downstreamDependencies, null, 2)}`
     }          
     `;
 
-    return observabilityAIAssistant.getContextualInsightMessages({
-      message: `I'm looking at an alert and trying to understand why it was triggered`,
-      instructions: dedent(
-        `I'm an SRE. I am looking at an alert that was triggered. I want to understand why it was triggered, what it means, and what I should do next.        
+      return observabilityAIAssistant.getContextualInsightMessages({
+        message: `I'm looking at an alert and trying to understand why it was triggered`,
+        instructions: dedent(
+          `I'm an SRE. I am looking at an alert that was triggered. I want to understand why it was triggered, what it means, and what I should do next.        
 
         The following contextual information is available to help me understand the alert:
         ${obsAlertContext}
 
         Be brief and to the point.
         Do not list the alert details as bullet points.
-        Do refer to the contextual information provided above when relevant.
+        Refer to the contextual information provided above when relevant.
         Pay specific attention to why the alert happened and what may have contributed to it.
         `
-      ),
-    });
+        ),
+      });
+    } catch (e) {
+      return observabilityAIAssistant.getContextualInsightMessages({
+        message: `I'm looking at an alert and trying to understand why it was triggered`,
+        instructions: dedent(
+          `I'm an SRE. I am looking at an alert that was triggered. I want to understand why it was triggered, what it means, and what I should do next.`
+        ),
+      });
```

Comment:
I wonder how this would help the assistant as an exception just happened and no details about the alert are passed.

Reply:
If an error occurs server-side the LLM will only have access to the alert details. It's not great but better than erroring out completely imo